src/app/api/manager/multiview-presets/route.ts

```diff
@@ -0,0 +1,48 @@
+import { NextRequest, NextResponse } from 'next/server';
+import {
+  getMultiviewPresets
+  // TODO Add this when possibility to update and add mv-presets are added
```

Comment:
Maybe these imports can be added later, when they are used instead of commenting them out? But that might be personal preference on my part for cleaner code